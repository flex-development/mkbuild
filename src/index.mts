/**
 * @file Package Entry Point
 * @module mkbuild
 */

export type * from '#interfaces/index'
export { default as make } from '#make'
export type * from '#types/index'
export * from '#utils/index'
