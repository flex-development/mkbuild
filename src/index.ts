/**
 * @file Package Entry Point
 * @module mkbuild
 */

export * from './config/constants'
export { default as defineBuildConfig } from './config/define'
export type { Config, Result } from './interfaces'
