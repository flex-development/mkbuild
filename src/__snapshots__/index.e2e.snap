// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:mkbuild > should expose public api 1`] = `
[
  "make",
  "defineBuildConfig",
  "formatToExt",
  "gitignore",
  "IGNORE_PATTERNS",
  "loadBuildConfig",
  "runnableTask",
  "TASK_PLUGIN_NAME",
]
`;
